environment.yml

```diff
@@ -8,10 +8,12 @@ dependencies:
 
 
   # Main data handling
+  - pyproj==3.4.0
   - iris
   - xarray
   - pandas
   - geopandas
+  - "shapely<2"
```

Comment:
Does this need to be in quote marks?  Not sure what difference it makes, but it might.